terraform/backend.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    bucket = "forest-test-state"
+    key = "terraform.tfstate"
+    region = "us-west-1"
+    endpoint = "nyc3.digitaloceanspaces.com"
+    skip_credentials_validation = true
+    skip_metadata_api_check = true
```

Comment:
Why?

Reply:
This can be useful in cases where Terraform cannot automatically determine the validity of the credentials or the metadata API availability. However, the user still wants to proceed with the Terraform operation. By skipping the validation checks, Terraform assumes that the provided credentials are valid and the metadata API is available and will proceed with the process.